indico/modules/events/requests/templates/event_request_details.html

```diff
@@ -76,15 +76,15 @@
     {% endif %}
 
     {% call form_footer(form) %}
-        <input class="i-button big highlight" type="submit" value="
+        <input id="send-request" class="i-button big highlight" type="submit" value="
             {%- if not req -%}
                 {%- trans -%}Send request{%- endtrans -%}
             {%- elif req.can_be_modified -%}
                 {%- trans -%}Modify request{%- endtrans -%}
             {%- else -%}
                 {%- trans -%}Send new request{%- endtrans -%}
             {%- endif -%}
-        ">
```

Comment:
Maybe we can use the opportunity to move this huge block to  `{% set ... %}` block so the definition of this HTML tag is less awful (we could then just use `value="{{ button_title }}"`)?